mysql-test/suite/sys_vars/t/legacy_user_identifier.test

```diff
@@ -0,0 +1,20 @@
+--source include/not_embedded.inc
+--source include/load_sysvars.inc
+
+set @@global.legacy_user_identifier = default;
+select @@global.legacy_user_identifier;
+set @saved_legacy_user_identifier = @@global.legacy_user_identifier;
```

Comment:
Doesn't it make sense to save the original value _before_ you set it in line 4?  Sure - you are likely running this with the default legacy_user_identifier, but it still looks weird.